doc/user/content/quickstarts/application.md

```diff
@@ -0,0 +1,231 @@
+
+Telecommunications companies use [antenna manufacturers’ key performance indicators (KPIs)](https://www.ericsson.com/en/reports-and-papers/white-papers/performance-verification-for-5g-nr-deployments) to run analytics. Let’s use these indicators as “performance” and use them to calculate different performance health statuses over an antenna.
+
+•	If the <strong> last-half-minute average performance</strong> is greater than 5, it is <span style='border-bottom:1px solid green;'>healthy</span>.
+
+•	If it is more than 4.75 but less than 5, it is <span style='border-bottom:1px solid #FFBA00;'>semi-healthy</span>.
+
+•	If it is less than 4.75, it is <span style='border-bottom:1px solid red;'>unhealthy</span>.
+
+Every time the operational analytics detects an unhealthy case during a brief period, it will deploy a whole set of helper antennas in the area. Once the area improves and becomes healthy, these antennas are deactivated to save money.
+
+Each antenna has a known fixed range capable of serving customers. Operations teams watch a map of green, yellow, or red circles (antenna’s range area and health status) to understand where the issues are.
+
+A traditional approach like batch processing demands extra time to detect any issue, exposing the clients to a bad experience. A real-time application should be able to keep customers, businesses, and operational people happy.
+
+## Conceptual Overview
+
+### Sources
+
+First, a [source](https://materialize.com/docs/sql/create-source/) needs to spin up. It could be Postgres, Redpanda, Kafka, S3, or other alternatives. It will define how to store, insert, and process in Materialize the performance events and antenna’s position on the map.
+
+{{< figure
+    src="https://materialize.com/wp-content/uploads/2022/04/Inner-Frame.png"
+    alt="Materialize sources"
+>}}
+
+### Parse
+
+The source defines how to parse the events schema.
+
+•	Postgres has tables with a defined schema.
+
+•	Kafka/Redpanda can have topics with or without a schema.
+
+For a quick start, let’s focus only on the Postgres case, where the source already defines a schema. If you want to know how it would look using Kafka/Redpanda without a schema, refer to [this repository](https://github.com/MaterializeInc/demos/tree/main/antennas-kafka).
+
+The first step while building an application using Postgres is making available the tables to consume from Materialize.
+It’s done through a process known as [Change Data Capture (CDC)](https://materialize.com/docs/sql/create-source/materialize-cdc/). As it refers to in the [integration guide](https://materialize.com/docs/integrations/cdc-postgres/), <i>“it allows you to track and propagate changes in a Postgres database to downstream consumers based on its Write-Ahead Log (WAL).”</i>
+
+From Materialize standpoint it requires creating a source:
+
+```sql
+    CREATE MATERIALIZED SOURCE IF NOT EXISTS antennas_publication_source
```

Comment:
The requirement for materializing a postgres source for multiple downstream views goes away in platform. Not sure if we should remove this now or later.

Reply:
I agree that not having it would be better, but removing the Materialization will end up in a `replication error slot` since it's using two Materialized views downstream.